App.js

```diff
@@ -1,28 +1,32 @@
-import React, {useEffect, useState} from 'react';
+import React, {useState} from 'react';
 import {
   View,
   Text,
   TextInput,
   StyleSheet,
   TouchableOpacity,
+  KeyboardAvoidingView,
+  Keyboard,
+  ScrollView,
 } from 'react-native';
+import Repo from './components/Repo';
 
 const App = () => {
   const [query, setQuery] = useState('');
   const [repo, setRepo] = useState([]);
 
-  useEffect(() => {
-    console.log(repo);
-  }, [repo]);
-
   const getReps = async () => {
-    try {
-      const api = 'https://api.github.com/search/repositories?q={';
-      let response = await fetch(api + query + '}');
-      let resp = await response.json();
-      setRepo(resp.items);
-    } catch (error) {
-      console.error(error);
+    if (query != '') {
+      alert('Click on Name to open Repository in browser.!');
+      Keyboard.dismiss();
+      try {
+        const api = 'https://api.github.com/search/repositories?q={';
+        let response = await fetch(api + query + '}');
```

Comment:
there is a possibility that some network error occurs and the response might not have items field. 
In such case the app will crash. you can add a null check to ensure that items are present before calling setRepo

```if resp && resp.items { ... ```

Reply:
ok